src/java/org/apache/cassandra/service/StorageService.java

```diff
@@ -1415,6 +1416,17 @@ public void setCompactionThroughputMbPerSec(int value)
                     value, oldValue);
     }
 
+    public Map<String, String> getCurrentCompactionThroughput()
```

Comment:
please javadoc this method, what will a user get in the map? Name of the method might reflect that units are MBs per second.